gadget/install/install.go

```diff
@@ -570,6 +571,8 @@ func MountVolumes(onVolumes map[string]*gadget.Volume, encSetupData *EncryptionS
 			errUnmount := sysUnmount(mntPt, 0)
 			if errUnmount != nil {
 				logger.Noticef("cannot unmount %q: %v", mntPt, errUnmount)
```

Comment:
should it also be indicated in the log that we are retrying with lazy?

Reply:
And add a small string here too